app/components/ChangeColorInput.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component, PropTypes } from 'react';
+import BaseComponent from './BaseComponent';
+import { CompactPicker } from 'react-color';
+
+export default class ChangeColorInput extends BaseComponent {
+
+  constructor() {
+    super();
+    this.state = {
+      displayColorPicker: false,
+      color: "#cccccc"
```

Comment:
since `props` are passed to the constructor, we can write `constructor(props)`, `super(props)` and `color: props.value` so that the picker's color is set to the node's color passed to the `value` prop by `UpdateNodeForm`.